Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM maven:3-jdk-8-onbuild
+
+ENV JVM_GC_OPTS="-XX:+UseG1GC -XX:MaxGCPauseMillis=200 -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled"
```

Comment:
I think the user could set the JAVA_OPTS environment variable and the Dockerfile could pass that directly to the java command.
```
CMD java $JAVA_OPTS -jar /usr/src/app/server/target/server-*.jar
```
and then the user can run the container by:
```
docker run -e JAVA_OPTS="-Xmx1024M -XX:+UseG1GC" "stackdriver-zipkin-server"
```

I think in this case, we don't need a separate entrypoint, and the user can pass in their JVM args. Would this approach work for your use case?